cmd/podman/load.go

```diff
@@ -81,7 +82,7 @@ func loadCmd(c *cliconfig.LoadValues) error {
 			defer os.Remove(outFile.Name())
 			defer outFile.Close()
 
-			inFile, err := os.OpenFile(input, 0, 0666)
+			inFile, err := os.OpenFile("/dev/stdin", 0, 0666)
```

Comment:
Shouldn't this be c.Input instead of the unconditional `"/dev/stdin"`

Reply:
No this is the case where the user did not specify a --input file, it will pull the image from stdin, create a temporary file and then send the temporary file to containers/storage.